styles.css

```diff
@@ -332,8 +332,8 @@ form input {
   display: grid;
   grid-template-columns: 1fr;
   margin: 0 auto;
-  padding: 0 20px;
-  gap: 8rem;
+  row-gap: 8rem;
+  padding: 0 60px;
 }
```

Comment:
Great design @Marlyn-Mayienga 🤝

- [ ] To make your design very responsive, you may please fix the section which is disappearing as you try to change the screen sizes.
![cards](https://user-images.githubusercontent.com/63917725/177222475-694579ec-bdc4-44ed-9565-58f2fd299c4f.PNG)